docs/book/stacks-and-components/component-guide/experiment-trackers/neptune.md

```diff
@@ -226,5 +203,19 @@ def my_step(
     ...
 ```
 
+### Neptune UI
+
+Neptune comes with a web-based UI that you can use to find further details about 
+your tracked experiments. Each pipeline run will be logged as a separate 
+experiment run in Neptune, which you can inspect in the Neptune UI:
+
+![Neptune UI](https://docs.neptune.ai/img/app/integrations/zenml.png)
```

Comment:
This image is 'not found' when I click on that URL...

Reply:
The docs haven't been updated on Neptune's end yet 🙈 
The image will be available once they are.

I linked directly to the image in Neptune's docs so that we don't have to create a new PR to ZenML whenever there's a UI update in Neptune.

However, I can also replace the image already being used here: https://github.com/zenml-io/zenml/blob/main/docs/book/.gitbook/assets/NeptuneUI.png

Let me know what you'd prefer

Reply:
Haha ok that makes sense. Will you let us know here when the URL is working? 

The only disadvantage of doing it this way is that at some point if you change the name or that file goes away, then our docs have a broken image, so we should be careful about that.

Reply:
My suggestion is to remove this entirely. Neptune UI screenshots tend to have poor shelf life ;)

Reply:
@strickvl - are you fine with having no screenshot?

If not, I'll replace [NeptuneUI.png](https://github.com/zenml-io/zenml/blob/main/docs/book/.gitbook/assets/NeptuneUI.png) with this one so that there's no dependency on Neptune's docs assets.

Reply:
That'd be fine with us to remove it. Less maintenance all round :)